scripts/Deploy.s.sol

```diff
@@ -16,29 +16,50 @@
 
 pragma solidity ^0.8.17;
 
-import "forge-std/Script.sol";
+import "../lib/forge-std/src/Script.sol";
 import "../relay/contracts/BonsaiRelay.sol";
 import "../contracts/BonsaiStarter.sol";
-import "bonsai-lib-sol/BonsaiCheats.sol";
+import "../lib/bonsai-lib-sol/src/BonsaiCheats.sol";
 
-contract Deploy is Script, BonsaiCheats {
+contract Relay is Script, BonsaiCheats {
     function run() external {
         uint256 relayPrivateKey =
             vm.envOr("RELAY_PRIVATE_KEY", uint256(0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80));
         vm.startBroadcast(relayPrivateKey);
 
         // Deploy a Relay contract instance
-        BonsaiRelayContract relayContract = new BonsaiRelayContract();
+        BonsaiRelay relayContract = new BonsaiRelay();
+        
+        IBonsaiRelay bonsaiRelay = IBonsaiRelay(address(relayContract));
+        console.logAddress(address(bonsaiRelay));
+
+        vm.stopBroadcast();
+    }
+}
+
+contract Starter is Script, BonsaiCheats {
```

Comment:
Nice! I also wrote my own `Deploy.s.sol` scrip this morning. Let's merge this one and I am going to follow up with a PR to this repo with learnings and improvements made while developing the Governance Showcase.

https://github.com/risc0/risc0/pull/695/files#diff-061b0d26d8e43b371c37f545270588d6fb59fa6118e65204f4c94f942abf37ae